lighthouse-core/audits/audit.js

```diff
@@ -125,7 +125,12 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
-
+    let auditDescription = audit.meta.description;
+    if (audit.meta.fallbackDescription) {
+      if (!result.rawValue || (typeof result.rawValue === 'number' && result.score < 100)) {
```

Comment:
@ev1stensberg I believe brendan intended this entire expression to be based on `score` the local variable since that already does the correct use of `result.rawValue` if a score isn't set